### ISSUE TITLE
Event Counts Fetching (NIP-45)

### ISSUE THREAD

**Issue by richard16611** [nostr-dev-kit/ndk#168]
As far as I can tell, NIP-45 for event count fetching is not supported right now. Is there a plan to integrate it in the near future?

**Comment by rodant** [comment]
I also read something similar here https://github.com/hoytech/strfry/issues/77#issuecomment-1821496030. But the nip is still in the nips repository and there are some implementations like in haven and nak. What is the consensus in NDK after more than a year?

**Comment by pablof7z** [comment]
We could certainly add support for it, it would be like a 20-line change or so, I haven't added it since it seems like such a useless command as it stands right now; there's a proposal to use HyperLogLog which would be far more useful, but, afaik, it's not really supported yet.

**Comment by rodant** [comment]
> We could certainly add support for it, it would be like a 20-line change or so, I haven't added it since it seems like such a useless command as it stands right now; there's a proposal to use HyperLogLog which would be far more useful, but, afaik, it's not really supported yet.

Yes, I agree the client implementation should be straight forward. I think the functionality can be useful, though. I have a concrete use case right now. Nevertheless your hint about HyperLogLog sounds very interesting, thanks for pointing at it.
